lib/testcontainers.ex

```diff
@@ -31,6 +31,10 @@ defmodule Testcontainers do
   @doc false
   def get_host(name \\ __MODULE__), do: wait_for_call(:get_host, name)
 
+  def get_docker_host(name \\ __MODULE__) do
```

Comment:
I guess this can be removed :)

Reply:
ah I see maybe why you added it now. But it wont work because you would need a handle_call for that event.